src/demo.rs

```diff
@@ -434,6 +438,83 @@ impl DemoManager {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct VoteCastEventWrapped {
```

Comment:
I'd probably make this type non-pub and place it in the sse-events module where it's used, since it doesn't have any utility outside of there.

Reply:
well its used in implementing the SerializableVoteEventContent trait,

`impl SerializableVoteEventContent for VoteCastEventWrapped {`

Reply:
line 471 or so